addons/mass_mailing/models/mailing.py

```diff
@@ -754,6 +757,35 @@ def _prepare_statistics_email_values(self):
     # TOOLS
     # ------------------------------------------------------
 
+    def _get_mailing_name(self, base_name, subject):
+        """Generate a name for the mailing.
+
+        :param base_name: base name to use
+        :param subject: subject of the mailing
+        """
+        base_name = base_name or ''
+        if subject and not base_name:
+            base_name = _("%s (Mailing created on %s)") % (
+                subject,
+                datetime.strftime(fields.datetime.now(), tools.DEFAULT_SERVER_DATE_FORMAT)
```

Comment:
Maybe `fields.Datetime.to_string(fields.Datetime.now())` is a bit more standard.

Reply:
@awa-odoo But with this syntax, the hour will be included in the name, and we want to have just the day :/ 

```
Test source (Mailing created on 2020-10-06 06:12:32)
```
Instead of

```
Test source (Mailing created on 2020-10-06)
```

Reply:
Maybe ?
```python
fields.Date.to_string(fields.Datetime.now().date())
```

Reply:
Yes I meant `fields.Date.to_string(fields.Date.today())`.

I'm a bit too used to Datetime so I typed it without really thinking.